lib/matplotlib/axes/_base.py

```diff
@@ -1577,7 +1577,9 @@ def axis(self, *args, emit=True, **kwargs):
             'on'     Turn on axis lines and labels. Same as ``True``.
             'off'    Turn off axis lines and labels. Same as ``False``.
             'equal'  Set equal scaling (i.e., make circles circular) by
-                     changing axis limits.
+                     changing axis limits. This is the same as
+                     ``ax.set_aspect('equal', adjustable='datalim')``.
+                     Explicit data limits may not be respected in this case.
             'scaled' Set equal scaling (i.e., make circles circular) by
                      changing dimensions of the plot box, then disable further
                      autoscaling.
```

Comment:
Maybe add an equivalent sentence for *'scaled'*  as well?

Reply:
done.